### DISCUSSION TITLE
Feature Request: Chart for Chakra v3

### DISCUSSION THREAD

**Discussion by imnash21** [chakra-ui/chakra-ui#8824] [Ideas] [answered]
### Description

Using **Recharts** or **ChartJS** then making it easy to use, adding props and a style inside a parent container.

### Problem Statement/Justification

The chart is one of the useful when building a dashboard (overview stats)

### Proposed Solution or API

<BarChart ... />
<LineChart ... />
Etc...

Or

<Chart type="bar" ... />
<Chart type="line" ... />
Etc...

**Comment by segunadebayo** [comment, marked as answer]
Charts shipped. Start here: https://chakra-ui.com/docs/charts/installation
